stdlib/websub/src/main/java/org/ballerinalang/net/websub/nativeimpl/GetSubscribers.java

```diff
@@ -57,24 +59,32 @@
 )
 public class GetSubscribers extends BlockingNativeCallableUnit {
 
+    private static final Logger log = LoggerFactory.getLogger(GetSubscribers.class);
+
     @Override
     public void execute(Context context) {
-        String topic = context.getStringArgument(0);
-        List<HubSubscriber> subscribers = Hub.getInstance().getSubscribers();
+        BValueArray subscriberDetailArray = null;
+        try {
+            String topic = context.getStringArgument(0);
+            List<HubSubscriber> subscribers = Hub.getInstance().getSubscribers();
 
-        final PackageInfo packageInfo = context.getProgramFile().getPackageInfo(WEBSUB_PACKAGE);
-        final StructureTypeInfo structInfo = packageInfo.getStructInfo(SUBSCRIPTION_DETAILS);
-        BValueArray subscriberDetailArray = new BValueArray(structInfo.getType());
+            final PackageInfo packageInfo = context.getProgramFile().getPackageInfo(WEBSUB_PACKAGE);
+            final StructureTypeInfo structInfo = packageInfo.getStructInfo(SUBSCRIPTION_DETAILS);
+            subscriberDetailArray = new BValueArray(structInfo.getType());
 
-        for (HubSubscriber subscriber : subscribers) {
-            if (subscriber.getTopic().equals(topic)) {
-                BMap<String, BValue> subscriberDetail = BLangVMStructs.createBStruct(
-                        structInfo, subscriber.getCallback(),
-                        subscriber.getSubscriptionDetails().get(SUBSCRIPTION_DETAILS_LEASE_SECONDS),
-                        subscriber.getSubscriptionDetails().get(SUBSCRIPTION_DETAILS_CREATED_AT));
-                subscriberDetailArray.append(subscriberDetail);
+            for (HubSubscriber subscriber : subscribers) {
+                if (topic.equals(subscriber.getTopic())) {
+                    BMap<String, BValue> subscriberDetail = BLangVMStructs.createBStruct(
+                            structInfo, subscriber.getCallback(),
+                            subscriber.getSubscriptionDetails().get(SUBSCRIPTION_DETAILS_LEASE_SECONDS),
+                            subscriber.getSubscriptionDetails().get(SUBSCRIPTION_DETAILS_CREATED_AT));
+                    subscriberDetailArray.append(subscriberDetail);
+                }
             }
+        } catch (Exception ex) {
+            log.error("Error occurred while getting available subscribers: " + ex.getMessage(), ex);
```

Comment:
Printing the message is not needed if we are passing the exception to the logger. If we do this there will be duplicates.